tests/test_modules/good/prorata_syntax.catala_en

```diff
@@ -0,0 +1,65 @@
+> Using Prorata_external as Ext
```

Comment:
Where is `Prorata_external` defined ? Is this really how you're supposed to use this builtin? Will it be the same for other builtins?

Reply:
https://github.com/CatalaLang/catala/blob/master/tests/test_modules/good/prorata_external.ml

Reply:
https://github.com/CatalaLang/catala/blob/master/tests/test_modules/good/prorata_external.catala_en

Reply:
(far from perfect but there is an HOWTO about how to handle it in its current state: https://github.com/CatalaLang/catala/blob/master/doc/devel/externals.md)